base-notebook/Dockerfile

```diff
@@ -24,7 +24,7 @@ USER root
 # (ARGS are in lower case to distinguish them from ENV)
 # Check https://github.com/conda-forge/miniforge/releases
 # Conda version
-ARG conda_version="4.10.0"
+ARG conda_version="4.10.1"
```

Comment:
This change is not actually that simple.
MiniForge has started using python 3.9, so with this change we will switch to python 3.9 as well.
You can see it in the tagging step :)

There is a small release note as well.
https://github.com/conda-forge/miniforge/releases

Reply:
Maybe we should wait at least till anaconda ships python 3.9 by default?

Reply:
I did not even imagine that the version of Python could change in a patch release. If you want to master the python version and avoid an unwanted change in the future I think we have to write a test. I can do it.

Reply:
For information, [Miniconda](https://docs.conda.io/en/latest/miniconda.html) (the previous installer is shipped also with Python 3.9 `Miniconda3-py39_4.9.2-Linux-x86_64.sh`.
What was applied here until now was to always use the latest versions of the software and to rely on tags if users want to use a previous version.

Reply:
I was also surprised by changing of the Python version in a patch release.

Yes, let's write a test, which detects the current python version and compares it with hardcoded constant - this way we will always know when python is updated.

Ok, let's write a test and update.

Reply:
Sure I will do that 👍

Reply:
Test implemented but I rolled back the version of Python to stick to `3.8.x`. We will update it later.